lib/tracer/tracer.js

```diff
@@ -17,6 +20,14 @@ export const Tracer = function () {
   this.maxArrayItemsToFilter = 20;
 };
 
+export const checkForNested = event => {
```

Comment:
This check was needed because some apps had multiple agents that would each try to use the same kadira info object. Now that we use async local storage, we shouldn't have to worry about other agents corrupting it.